recipes-bsp/opensbi/opensbi_%.bbappend

```diff
@@ -1,2 +1,8 @@
 # Support fdt drivers for AE350
 SRCREV:ae350-ax45mp = "22f38ee6c658a660083aa45c4ec6c72f66a17260"
+
+BRANCH:visionfive2 = "JH7110_VisionFive2_devel"
+SRC_URI:visionfive2 = "git://github.com/starfive-tech/opensbi.git;protocol=https;branch=${BRANCH}"
```

Comment:
how different is this fork from upstream? is there a plan to upstream the changes can you comment on that ?